taxcalc/behavior.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+class Behavior(object): 
+
+	def __init__(self, calcX, calcY, elast_wrt_atr = 0.4, inc_effect = 0.15):
+		self.calcX = calcX
+		self.calcY = calcY
+		self.elast_wrt_atr = elast_wrt_atr
+		self.inc_effect = inc_effect
+
+	@property
+	def calc(self):
```

Comment:
Made behavior into a function with 12118db 

Now you can run calc_y_behavioral = behavior(calc_x, calc_y)

This also allows you to compare calc_y without behavior to calc_y_behavioral, which is a useful feature.